internal/service/ec2/ec2_placement_group.go

```diff
@@ -210,5 +211,11 @@ func resourcePlacementGroupCustomizeDiff(_ context.Context, diff *schema.Resourc
 		}
 	}
 
+	if diff.Id() == "" {
+		if spreadLevel, strategy := diff.Get("spread_level").(string), diff.Get("strategy").(string); spreadLevel == "" && strategy == ec2.PlacementGroupStrategySpread {
+			return diff.Clear("spread_level")
+		}
+	}
```

Comment:
Clearing the diff if `spread_level` is not set but `strategy` is set to `spread`, wasn't sure if this was best way to achieve desired outcome or if there's a better way. Still getting feet wet with this

Open to suggestions, improvements, and feedback. Thanks!

Reply:
This code isn't necessary. Setting `spread_level` to _Computed_ is sufficient.